doc/source/conf.py

```diff
@@ -28,18 +29,23 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.append(os.path.abspath('../sphinxext'))
 
-# -- General configuration -----------------------------------------------------
+# -- General configuration ----------------------------------------------------
 
 # We load the nibabel release info into a dict by explicit execution
 rel = {}
-execfile('../../nibabel/info.py', rel)
+try:
```

Comment:
Omit this and just use the `with open` branch on both 2 and 3?

Use `os.path.join` for Windows compatibility?